src/style.css

```diff
@@ -2,11 +2,19 @@ body {
     font-family: 'Spartan', sans-serif;
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
+    padding: 0;
+    margin: 0;
+
 }
 
 div#root {
     display: flex;
     justify-content: center;
+    padding-top: 5em;
+    position: fixed;
+    width: 100%;
```

Comment:
The calculator width should be 700px ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/700Wide.png)